README.md

```diff
@@ -41,6 +41,16 @@ Please see the
 [Guest Science Resources](https://www.nasa.gov/content/guest-science-resources)
 page for information on guest science capabilities on Astrobee.
 
+## Prerequisites
+
+Before building the Astrobee Android project, ensure your system meets the following requirements:
+
+- **Operating System**: Ubuntu 16.04 (Xenial Xerus) is needed for compatibility with `ros-kinetic-rosjava` dependencies.
```

Comment:
@rgarciaruiz should comment on our Ubuntu version strategy.

The Astrobees on ISS indeed currently run Ubuntu 16.04 on the MLP and LLP, but there is a plan to upgrade them to Ubuntu 20.04, hopefully later this year. Also, the `astrobee_android` code of course runs on the HLP under Android (not Ubuntu), so to some extent it should be isolated from caring about the Ubuntu version running on the other processors.

It's possible that you have two viable options for your `astrobee_android` development:

1. Develop under Ubuntu 20.04 and use `ros-noetic-rosjava`. If this works, it would be preferred in terms of being easier to maintain long-term and more compatible with upcoming Astrobee upgrades.
2. Develop under Ubuntu 16.04 and use `ros-kinetic-rosjava`, as you are suggesting. This would be the highest-fidelity test if you are targeting an ISS activity this summer, but I think it's otherwise not recommended for most users.

I'm not aware that anybody has tested option 1 -- @rgarciaruiz would be most likely to know. It's possible there is no such version of `rosjava`, or that using it might require some cascading changes elsewhere in the system. One potential concern would be that a `noetic` version of `rosjava` might have incompatibilities with the `kinetic` version of ROS1 running on the MLP/LLP at the network protocol level. But I think this is very unlikely because the ROS1 network protocol has been quite stable for a while, and we are using it in a limited, straightforward way.

(I'm sorry I can't be more definitive about this! But I think we should really mention this issue if we are going to update documentation of the system requirements.)

Reply:
There are some dependencies that are not the same between ROS versions. I closed out an issue after I switched from 20.04 to 16.04 in this issue https://github.com/nasa/astrobee_android/issues/44#issue-1679160169

Reply:
That's fine. However, if that is the case it might be worthwhile to add instructions on how to build the `ros-kinetic-rosjava` package from source for ROS Noetic since `ros-noetic-rosjava` does not exist yet. My other updates are also helpful so I can make another pull request with those changes if that sounds alright to you.

Reply:
Yes, thanks for taking the time to push your changes as an upstream PR! I shouldn't be the primary reviewer on this PR, but we appreciate your efforts!

I guess a quick interim fix would be to keep your note about the current build instructions assuming Ubuntu 16.04, but also mention in the same place that Ubuntu 16.04 support will soon be phased out as we migrate to Ubuntu 20.04, and reference your earlier issue describing the current (temporary) issue with `astrobee_android` under 20.04. Then it would tell the whole story. Thanks!

Reply:
Sounds good. Should I open an issue regarding the dependencies, or do you guys already have one open?